README.md

```diff
@@ -37,8 +49,8 @@ install.packages("infer")
 ```
 
 To install the developmental version of `infer`, make sure to install
```

Comment:
@simonpcouch "Developmental" is used in two different contexts here. Maybe change infer-dev.netlify.com to be "bleeding edge" or something similar? Maybe `master` could correspond to "developmental stable" and `develop` could be "experimenting"?